code/modules/reagents/reagents/reagents_medical.dm

```diff
@@ -1240,7 +1240,7 @@ var/global/list/charcoal_doesnt_remove=list(
 	reagent_state = REAGENT_STATE_LIQUID
 	color = "#1A1A1A" //rgb: 26, 26, 26
 	density = 2.46
-	specheatcap = 12439.3 //Good fucking luck
+	specheatcap = 0.124393
```

Comment:
phalanximine goes from one of the best heat absorbers to one of the worse
![gosling agony](https://github.com/vgstation-coders/vgstation13/assets/67024428/7a3d2a08-6e92-4faf-bb71-ee51df81f64d)

Reply:
take your pick:
1. use leporazine or any of those with an arbitrarily large number specheatcap
2. code a new reagent designed specifically to have an even higher specheatcap (maybe have it require some exotic reagents/materials so it's a neat reward for acquiring those)
3. increase the arbitrarily large number define.

Reply:
4. new reagent with a negative specheatcap

Reply:
can you increase the arbitrarily large specheatcap? add a few zeroes to it